src/components/AutoInputBinLocation/index.tsx

```diff
@@ -33,6 +39,10 @@ export default function ({data, selectedData, initValue = ''}: Props) {
           value={query}
         />
       </ModalSelector>
+      {query != '' ?
```

Comment:
You don't need to check if its not equal to empty string and youse , you can leave only the `{query ? ...`